.github/workflows/release-please.yml

```diff
@@ -27,4 +27,6 @@ jobs:
               { "type": "fix", "section": "Bug Fixes" },
               { "type": "build", "scope": "deps", "section": "Dependency Updates" }
             ]
-          release-type: go
+          release-type: simple
```

Comment:
What does the change from `go` to `simple` mean?  When looking at https://github.com/googleapis/release-please/blob/main/docs/customizing.md#strategy-language-types-supported , this is the difference:

| release type        | description |
|---------------------|---------------------------------------------------------|
| `go`                | A repository with a CHANGELOG.md |
| `simple`            | [A repository with a version.txt and a CHANGELOG.md](https://github.com/googleapis/gapic-generator) |

Reply:
In practice, this results in the same process. I can postpone this change to a follow-up PR if you prefer. As this is not a standard Go project, providing a module to consumers in other projects, I think `simple` is more describing than `go`.

Reply:
I see, I'm OK with having it in this PR, I was just curious.